scripts/send_to_bot.py

```diff
@@ -34,8 +29,8 @@ def post_comment(pr_id, comment_body):
 "{comment_body}"
 "\n```\n"
 "Please fix the error(s) and push again."
-).format(build_id=BUILD_ID, comment_body=test_result)
+).format(build_id=os.environ.get('TRAVIS_BUILD_ID'), comment_body=test_result)
 
 # If it's a PR, post a comment on it
-if PR_NUMBER != "false":
-  post_comment(PR_NUMBER, comment)
+if os.environ.get('TRAVIS_PULL_REQUEST') != "false":
+  post_comment(os.environ.get('TRAVIS_PULL_REQUEST'), comment)
```

Comment:
Save as above. Here we're duplicating effort, and if we need to change the variable name later it's more effort to change.